azure-pipelines.yml

```diff
@@ -167,8 +167,18 @@ jobs:
           test_suites:
               commoncode: |
                 bin/py.test -vvs tests/commoncode/test_codec.py \
+                    tests/commoncode/test_command.py \
+                    tests/commoncode/test_date.py \
+                    tests/commoncode/test_filetype.py \
+                    tests/commoncode/test_fileset.py \
                     tests/commoncode/test_fileutils.py \
-                    tests/commoncode/test_functional.py
+                    tests/commoncode/test_functional.py \
+                    tests/commoncode/test_ignore.py \
+                    tests/commoncode/test_paths.py \
+                    tests/commoncode/test_saneyaml.py \
+                    tests/commoncode/test_timeutils.py \
+                    tests/commoncode/test_urn.py \
+                    tests/commoncode/test_version.py
```

Comment:
At this stage if all test modules are importable you may be able to use `bin/py.test -vvs tests/commoncode` more simply?